docker.sh

```diff
@@ -1,10 +1,8 @@
 #!/usr/bin/env bash
 set -e
+# Environment variables in common with the Docker.ps1 comes from the docker.env
```

Comment:
nit: since you need to change something, I'd move this comment in the first commit, or just remove it